### ISSUE TITLE
Axis won't render

### ISSUE THREAD

**Issue by mikebronner** [shutterstock/rickshaw#467]
I'm going through the tutorials, but am unable to get the x-Axis to render. 

```
    <div id="gl_whs_chart_container">
        <div id="y_axis"></div>
        <div id="chart"></div>
    </div>

    <script>
        var graph = new Rickshaw.Graph.Ajax( {
            element: document.querySelector("#chart"),
            renderer: 'line',
            dataURL: 'ajax.php',
            width: 580,
            height: 250,
            series: [ {
                name: 'Ambient Temperature'
            } ]
        } );
        var x_axis = new Rickshaw.Graph.Axis.Time( { graph: graph } );
        var y_axis = new Rickshaw.Graph.Axis.Y( {
            graph: graph,
            orientation: 'left',
            tickFormat: Rickshaw.Fixtures.Number.formatKMBT,
            element: document.getElementById('y_axis'),
        } );
        graph.request();
    </script>
```

The ajax.php returns the following dataset:

```
[{"color":"steelblue","name":"Ambient Temperature","data":[{"x":1407978270,"y":22.47},{"x":1407978353,"y":22.53},{"x":1407978353,"y":22.53},{"x":1407978415,"y":22.63},{"x":1407978523,"y":22.51},{"x":1407978524,"y":22.51},{"x":1407978546,"y":22.5},{"x":1407978562,"y":22.41},{"x":1407978571,"y":22.35}]}]
```

The output looks like this:
![Bar With Axis Labels](http://share.genealabs.com/download/C-Xq1XLeyGSXGTRe2Rxw6HGNvhyEdUTB_52wqNL0Wp8G9ieAoEVYZ9h--hbuOLClKKKqm53q-Ws6mUaTBwrhvw)

You can see the numbers to the bottom left, those have something to do with the axis labels, but not sure why its not showing the time.
